CI failure: Module 'mirroring' is not enabled

We're seeing the following test flake somewhat regularly now:

```
=== RUN   TestParsePeerList/ok
--- PASS: TestParsePeerList (0.00s)
    --- PASS: TestParsePeerList/error (0.00s)
    --- PASS: TestParsePeerList/statusSet (0.00s)
    --- PASS: TestParsePeerList/ok (0.00s)
=== RUN   TestMirroring
    mirror_workflow_test.go:62: 
        	Error Trace:	mirror_workflow_test.go:62
        	Error:      	Received unexpected error:
        	            	rados: ret=-95, Operation not supported: "Module 'mirroring' is not enabled (required by command 'fs snapshot mirror enable'): use `ceph mgr module enable mirroring` to enable it"
        	Test:       	TestMirroring
--- FAIL: TestMirroring (12.85s)
```

I suspect that the API that enables modules doesn't guarantee callers will see the module as enabled at return time, thus producing a race between us activating the module and then immediately trying to use it before the mgr is ready to serve requests that make use of the modue.

--- a/cephfs/admin/mirror_workflow_test.go
+++ b/cephfs/admin/mirror_workflow_test.go
@@ -41,6 +41,7 @@ func TestMirroring(t *testing.T) {
 		assert.NoError(t, err)
 	}()
 	require.NoError(t, err)
+	time.Sleep(500 * time.Millisecond) // TODO: improve this
 
 	smadmin1 := fsa1.SnapshotMirror()
 	err = smadmin1.Enable(fsname)
@@ -49,6 +50,7 @@ func TestMirroring(t *testing.T) {
 		err := smadmin1.Disable(fsname)
 		require.NoError(t, err)
 	}()
+	time.Sleep(500 * time.Millisecond) // TODO: improve this
 
 	fsa2 := newFSAdmin(t, mirrorConfig())
 	err = fsa2.EnableMirroringModule(noForce)

